packages/astro/src/runtime/client/dev-overlay/ui-library/window.ts

```diff
@@ -30,7 +30,7 @@ export class DevOverlayWindow extends HTMLElement {
 					font-family: ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
 					color: rgba(204, 206, 216, 1);
 					position: fixed;
-					z-index: 9999999999;
+					z-index: 999999999;
```

Comment:
Might be nice to extract each `z-index` we need to a custom property at the root of the dev overlay component? Then this would be slightly easier to manage in one place.

If there's a good reason you're not already doing that, ignore me!